### ISSUE TITLE
criterion_segment  = torch.nn.BCEWithLogitsLoss() # non convergence？I know the answer!

### ISSUE THREAD

**Issue by DCC-lzhy** [mengcius/Surface-Defect-Detection#3]
https://github.com/mengcius/Surface-Defect-Detection/blob/1f42670e55d9ab0d328e139faa17a33c9149ec32/train_segment.py#L48

https://github.com/mengcius/Surface-Defect-Detection/blob/1f42670e55d9ab0d328e139faa17a33c9149ec32/models.py#L74

where you use the layer activate function is relu，but becwithlogitsloss has the sigmoid, that's error, it will train hardly, that's why non convergence.

as usual, the last layer of segment model is just a conv and no act.

in the paper, he conclused ce loss is better than mse.
